components/script/dom/document.rs

```diff
@@ -4248,6 +4247,8 @@ impl DocumentMethods for Document {
             vec![]
         };
 
+        // FIXME: Why are we silencing errors here? Is it really okay to return
```

Comment:
I'm not really involved with this project anymore, and please forgive me if I'm just making unproductive noise, but I notice this observation here wasn't brought up in #34849 and I never did find out whether or not it was meaningful.

Reply:
Good point. There are more cases where we are silencing errors where it might not make sense. This might happen if, for instance, the resource thread ended unexpectedly. We need to think more deeply about what should happen in those kind of cases.